src/controllers/dynakube/dynakube_controller.go

```diff
@@ -301,6 +276,37 @@ func (controller *DynakubeController) reconcileDynaKube(ctx context.Context, dkS
 	dkState.Update(upd, "dynakube phase changed")
 }
 
+func (controller *DynakubeController) reconcileOneAgent(ctx context.Context, dkState *status.DynakubeState) (err error) {
+	if dkState.Instance.HostMonitoringMode() {
+		upd, err := oneagent.NewOneAgentReconciler(
```

Comment:
introduces a dependency on an oneagent reconciler instance which cannot be mocked / faked for testing.

Reply:
Out of scope scope of this PR.

Reply:
Out of scope of this PR.